website/js/index.js

```diff
@@ -416,12 +430,99 @@ function isEmptyDiff(value) {
     return value === null || value.length < 1;
 }
 
+// attempts to parse the given date string using a variety of formats,
+// returning the formatted result as something like '08 September 2016'.
+// just returns the input if every pattern fails to match
+function normalizeDateFieldDisplay(value) {
+    // extend this if there are more formats in the future
+    const formats = ["MM/DD/YYYY", "YYYY-MM-DD"];
+
+    for (let i = 0; i < formats.length; i++) {
+        const q = moment(value, formats[i]);
```

Comment:
Considering that 'q' is never reassigned, I think it's fine to leave it const? Const doesn't indicate anything about the immutability of the object itself, just that the association between the symbol 'q' and that value can't be changed once assigned.